plugins/mongodb/mongodbembed.cpp

```diff
@@ -130,6 +130,98 @@ namespace mongodbembed
         }
     }
 
+    /**
+     * @brief Helper method for converting an Extended JSON structure into standard JSON
+     * 
+     * @param result Result string for appending results
+     * @param start Pointer to beginning of structure
+     * @param row Pointer to the last place where there was standard JSON
+     * @param lastBrkt Pointer to the begining of the EJSON structure
+     * @param depth Depth of the structure
+     */
+    void convertEJSONTypes(std::string &result, const char * &start, const char * &row, const char * &lastBrkt, int &depth) 
+    {
+        while (*start && *start != '$') start++;
+        auto end = start;
+        while (*end && *end != '\"') end++;
+        auto key = std::string(start, end - start); // Get datatype
+        result += std::string(row, lastBrkt - row); // Add everything before we went into nested document
+        if (key == "$regularExpression") // Some data types are unsupported as they are not straightforward to deserialize
+        {
+            UNSUPPORTED("Regular Expressions"); // TO DO handle unsupported types by not throwing an exception.
```

Comment:
I cannot view that issue. "It may have been deleted or you don't have permission to view it." Is what I am seeing.

Reply:
No problem, it's not assigned to you (unless you want to take it on), we'll change its visibility to community soon.

Reply:
I'll keep an eye on it then. Obviously anything involving this plugin I would be happy to help with in the future, and since I had such a positive experience with the platform I will definitely be looking for more ways to contribute.

Reply:
I have made that JIRA ticket public